installer/scripts/omsadmin.sh

```diff
@@ -161,6 +161,9 @@ usage()
     echo
     echo "Detect if omiserver is listening to SCOM port:"
     echo "$basename -o"
+    echo
+    echo "Show the configured process limit in /etc/limits.conf for user $AGENT_USER."
```

Comment:
Use the `PROC_LIMIT_CONF` variable here, especially since this is not the correct path to look for the limit in.

Reply:
Got it.